### DISCUSSION TITLE
clear data after running so-test

### DISCUSSION THREAD

**Discussion by dlazure** [Security-Onion-Solutions/securityonion#6527] [General] [answered]
Hi,

How do I clean my data after running so-test?

it shows a lot of alerts from the test and I would like to clean the data

thank you

**Comment by dougburks** [comment, marked as answer]
Take a look at `so-elastic-clear`. You might want to try it on a test box to make sure it does what you want it to do.

**Reply by dlazure** [reply]
exactly what I needed, 

Thank you Doug,

**Reply by dougburks** [reply]
Added to documentation:
https://docs.securityonion.net/en/2.3/elasticsearch.html#clearing
